File: EX01.java
import javax.swing.JOptionPane;

public class EX01{
	public static void main(String[] args){
		float som,num,media;
		int cont;

		som = 0;
		cont = 0;

		do{
			num = Float.parseFloat(JOptionPane.showInputDialog(null,"Digite um número"));
			som = som + num;
			cont++;
		}while(cont < 20);
		
		media = som / cont;
		JOptionPane.showMessageDialog(null,"A média dos valores deu " + media);
	}
}

File: Ruby/EX01.rb
def media(cont)
    som = 0
    i = 0

    while i < cont
        puts "digite um numero:"
        num = gets.chomp.to_i
        som = som + num 
        i += 1
    end

    med = som / cont
    return puts "a média desses valores é " + med.to_s

end

media(5)